src/Maestro/Client/src/MaestroApiCredential.cs

```diff
@@ -54,23 +57,78 @@ public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext _, Canc
         internal static MaestroApiCredential CreateUserCredential(string barApiBaseUri)
         {
             string appId = EntraAppIds[barApiBaseUri.TrimEnd('/')];
+            var requestContext = new TokenRequestContext(new string[] { $"api://{appId}/{USER_SCOPE}" });
 
-            // This is a usual credential obtained against an entra app through a browser sign-in
-            var credential = new InteractiveBrowserCredential(new InteractiveBrowserCredentialOptions
+            // This is a usual configuration for a credential obtained against an entra app through a browser sign-in
```

Comment:
We could just create these credential options inside `GetCredentialFromCache`?

Reply:
True, now they don't need to be outside. Thanks, will improve